openedx_events/content_authoring/data.py

```diff
@@ -12,6 +12,19 @@
 
 import attr
 from opaque_keys.edx.keys import CourseKey, UsageKey
+from opaque_keys.edx.locator import LibraryLocatorV2, LibraryUsageLocatorV2
+
+
+@attr.s(frozen=True)
+class CourseData:
+    """
+    Attributes defined for Open edX Course object.
+
+    Arguments:
+        course_key (CourseKey): identifier of the Course object.
+    """
+
+    course_key = attr.ib(type=CourseKey)
```

Comment:
In the learning subdomain we have a bit more info about the course, shouldn't be handy here? Can you explain why is this enough?

Reply:
I like this minimalistic approach, to just use what's necessary. Thanks!